test/pummel/test-fs-watch-non-recursive.js

```diff
@@ -38,13 +38,12 @@ const filepath = path.join(testsubdir, 'watch.txt');
 
 fs.mkdirSync(testsubdir, 0o700);
 
-// Need a grace period, else the mkdirSync() above fires off an event.
+const watcher = fs.watch(testDir, { persistent: true }, (event, filename) => {
+  assert.strictEqual(filename, 'testsubdir');
+});
+setTimeout(function() {
```

Comment:
Optional nit: We favor arrow functions for callbacks in general, so maybe switch to that here and in the existing call in the `setTimeout()` that follows.